modules/bridge/qb/server.lua

```diff
@@ -60,7 +60,16 @@ local function setupPlayer(Player)
 	end)
 
 	QBCore.Functions.AddPlayerMethod(Player.PlayerData.source, "GetItemByName", function(item)
-		return Inventory.GetItem(Player.PlayerData.source, item, nil, false)
+		local search = exports.ox_inventory:Search(Player.PlayerData.source, 'slots', item)
+		if search == {} then
```

Comment:
Search will never be equal to {} (two different tables with their own hashes). You can just check `next(search)`.